.circleci/config.yml

```diff
@@ -131,10 +131,45 @@ jobs:
             - /home/circleci/conda
           key: conda-deps-v1-{{ checksum "python/environment.yml" }}
 
+  spark-3-tests:
+    <<: *setup_base
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - conda-deps-v1-{{ checksum "python/environment.yml" }}
+      - *install_deps
+      - run:
+          name: Change Spark version
+          environment:
+          command: |
+            echo "3.0.0-SNAPSHOT" > spark-version.txt
+      - run:
+          name: Run Scala tests
```

Comment:
Seems like we should bump the pyspark version and run Python tests as well

Reply:
PySpark isn't released in the nightly snapshots. If we wanted to test against PySpark 3.0, we'd have to use one of the preview releases (eg. https://archive.apache.org/dist/spark/spark-3.0.0-preview2/), which we are currently ahead of.